vernac/vernacstate.mli

```diff
@@ -18,11 +18,27 @@ module Parser : sig
 
 end
 
+module Stack : sig
```

Comment:
Might be better to call it `ProofStack` or `LemmaStack` since it's not `Lemmas.Stack` anymore yet it's not an arbitrary `Stack`

Reply:
Indeed, which one do you prefer?

Reply:
LemmaStack probably makes more sense.

Reply:
Done